.travis.yml

```diff
@@ -29,6 +29,8 @@ matrix:
       env: SYMFONY_VERSION=3.0.*
     - php: 7.0
       env: FOSUSERBUNDLE_VERSION=2.0.*
+    - php: 7.1
+      env: DEPENDENCIES=beta
```

Comment:
`STABILITY` means more sens according to the action below. Could you please change that?